controller/konnect/reconciler_controlplaneref.go

```diff
@@ -230,3 +211,17 @@ func conditionMessageReferenceKonnectAPIAuthConfigurationInvalid(apiAuthRef type
 func conditionMessageReferenceKonnectAPIAuthConfigurationValid(apiAuthRef types.NamespacedName) string {
 	return fmt.Sprintf("referenced KonnectAPIAuthConfiguration %s is valid", apiAuthRef)
 }
+
+func controlPlaneRefToString(cpRef configurationv1alpha1.ControlPlaneRef) string {
```

Comment:
Why not add a `String()` method on the `ControlPlaneRef` type?

Reply:
Yeah, it makes sense. 👍 https://github.com/Kong/kubernetes-configuration/pull/230